templates/dashboard.html

```diff
@@ -3,11 +3,13 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- Refresh every 30 seconds -->
+    <meta http-equiv="refresh" content="30"> 
     <title>Leaderboard</title>
     <link rel="stylesheet" href="{{ url_for('static', filename='css/dashboard.css') }}">
 </head>
 <body>
-    <div class="container">
```

Comment:
Might still be necessary? Layout looks stretched rn

Reply:
Yeah accidentally deleted that, looks like that css class does exist, mb :/